doc/source/tutorials/Q-Factor.ipynb

```diff
@@ -0,0 +1,464 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e62a21ae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "diam, S_V, S_T = Q.Q_circle()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e7a62a1",
+   "metadata": {},
+   "source": [
+    "Use a polar plane to check the results: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb1e2ce2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(subplot_kw={'projection': 'polar'})\n",
+    "resonator.plot_s_polar(ax=ax, label=\"RLC Resonator\", ls='', marker='x', ms=5)\n",
+    "fitted_network.plot_s_polar(ax=ax, label=\"Fitted Model\", lw=2)\n",
+    "ax.plot(np.angle(S_V), np.abs(S_V), 'ko')\n",
+    "ax.plot(np.angle(S_T), np.abs(S_T), 'ko')\n",
+    "ax.text(np.angle(S_T), 0.8*np.abs(S_T), '$S_T$')\n",
+    "ax.text(np.angle(S_V), 1.1*np.abs(S_V), '$S_V$')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3441f1f9",
+   "metadata": {},
+   "source": [
+    "## Bandwidth\n",
+    "Another definition of the Q-factor is the frequency-to-bandwidth ratio of a transmission resonator [6]:\n",
+    "\n",
+    "$$\n",
+    "Q_L = \\frac{\\omega_r}{\\Delta \\omega} = \\frac{f_r}{\\Delta f} \n",
+    "$$\n",
+    "\n",
+    "where $f_r$ is the resonant frequency, $\\Delta f=BW$ is the resonance width or  _fractional bandwidth_ BW. Note that this definition is only an approximation, which can be not accurate when there is significant leakage (see [4] for more information). The definition of the bandwidth also depends on the resonator type:\n",
+    "\n",
+    "* transmission:  bandwidth, also known as _full width at half maximum_ (FWHM), is the bandwidth over which the dissipated power is half the maximum value at the resonant frequency (3 dB below the max value).\n",
+    "* reflection: while it is possible to define the bandwidth as the 3-dB points below the max reflection coefficient [6], a more proper definition of the bandwidth consists in measuring the difference between frequencies $f_1$ and $f_2$, which belong to the two points inclined by 45 degrees on each side of the centerline from $S_T$ to the origin (see [4,8] for additional details).\n",
+    "\n",
+    "Using the previous relation, the bandwith BW can also be obtained from the `Qfactor` class via the `.BW` parameter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c3217598",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BW = Q.BW\n",
+    "print(f'Bandwidth: {BW} Hz')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "240d7ea3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots()\n",
+    "resonator.plot_s_db(label='Parallel RLC ', lw=2, ax=ax)\n",
+    "ax.axvspan(xmin=Q.f_L-Q.BW/2, xmax=Q.f_L+Q.BW/2, alpha=0.3, label='Bandwidth')\n",
```

Comment:
Numpy is not happy here, a warning is printed. I think the problem is that `Q.BW` is a list and should be a number. So maybe try `Q.BW[0]`?